spec/services/google/client_service_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::ClientService, type: :service do
+  context 'when querying a simple keyword' do
+    it 'returns an HTTParty Response', vcr: 'google_search' do
+      result = described_class.new(keyword: FFaker::Lorem.word).call
+
+      expect(result).to be_an_instance_of(HTTParty::Response)
+    end
+
+    it 'queries Google Search', vcr: 'google_search' do
+      path = described_class.new(keyword: FFaker::Lorem.word).call.request.path
+
+      expect(path.to_s).to start_with('https://www.google.com/search')
+    end
+  end
+
+  context 'when google returns an HTTP error' do
+    it 'returns false', vcr: 'google_warn' do
+      result = described_class.new(keyword: FFaker::Lorem.word).call
+
+      expect(result).to be_falsey
```

Comment:
we can compare exact value, falsey means either of `nil or false`(which are falsy values in ruby)

```
   expect(result).to eq(false)
```

Reply:
Understood! Good to know! :)